components/hab/src/cli/hab/sup.rs

```diff
@@ -190,8 +185,8 @@ pub struct SupRun {
     // TODO (DM): Currently there is not a good way to use `parse_peer` when deserializing. Due to
     // https://github.com/serde-rs/serde/issues/723. There are workarounds but they are all ugly.
     // This means that you have to specify the port when setting this with a config file.
```

Comment:
we should remove this comment

Reply:
Addressed in commit 8930f52d98b666253fb494c288a5c2cdb3bf686c.